src/patterns/OcAutocomplete.vue

```diff
@@ -114,10 +118,10 @@ export default {
       default: 5,
     },
     /**
-     * The selected element
+     * Select what values an object is to be found by
      */
-    value: {
```

Comment:
why was value removed? this is the model which shall hold the selected element.
The parent component needs this to get informed about selection.

please bring it back. THX

Reply:
The `input` value does not reflect the `output` value, as we won't be able to display an object inside the `<input />` element. There is the `@select` that returns the value once selected. I still can wire up the input, should I?